### ISSUE TITLE
Typescript error vscode FormKitComponent

### ISSUE THREAD

**Issue by gunter423** [formkit/formkit#900]
### Reproduction

https://formkit.link/e6e026b413c717931d3dc9524a174080

### Describe the bug

Hi,

Look at the screenshot I don't know why there's this typescript error on vscode ?

<img width="966" alt="Screenshot 2023-08-18 at 15 52 58" src="https://github.com/formkit/formkit/assets/24797165/513fe49a-698f-4bd5-91a4-58a126ac04c8">

### Environment

• OS: MacOS Ventura
• Browser chrome

**Comment by justin-schroeder** [comment]
Gonna need a bit more details on this one. Can you provide your implementation of the component?

**Comment by njoswald** [comment]
It's not the same error, but I am also getting a TS error relating to the Multi-step plugin for VS Code. My #stepNext override is returning a TS error in the most recent update. 
![image](https://github.com/formkit/formkit/assets/119365022/e155a3b8-264a-4d0a-8adb-bc81ce16fbda)
This is the exact same example from [https://formkit.com/plugins/multi-step](https://formkit.com/plugins/multi-step)

**Comment by GustavoFenilli** [comment]
There is an updated version that fixes errors with typings for the multi-step that came with 0.18.

**Comment by gunter423** [comment]
> Gonna need a bit more details on this one. Can you provide your implementation of the component?

@justin-schroeder yes paste this : https://formkit.link/e6e026b413c717931d3dc9524a174080 into vscode :)

**Comment by justin-schroeder** [comment]
> @justin-schroeder yes paste this : https://formkit.link/e6e026b413c717931d3dc9524a174080 into vscode :)

I meant things like — your version of formkit, your version of formkit pro, your version of vue etc. This error should not happen if you have 0.114.x and beta 0.18.x installed.
